src/vs/workbench/electron-sandbox/desktop.main.ts

```diff
@@ -207,6 +205,19 @@ export class DesktopMain extends Disposable {
 			logService.trace('workbench#open(): with configuration', safeStringify({ ...this.configuration, nls: undefined /* exclude large property */ }));
 		}
 
+		// Default Account
+		const defaultAccountService = this._register(new DefaultAccountService());
+		serviceCollection.set(IDefaultAccountService, defaultAccountService);
+
+		// Policies
+		let policyService: IPolicyService = new NullPolicyService();
+		if (this.configuration.policiesData) {
+			const policyChannel = new PolicyChannelClient(this.configuration.policiesData, mainProcessService.getChannel('policy'));
```

Comment:
I think you should create AccountPolicyService always. `this.configuration.policiesData` is meant for polices read from registry or config profiles.